shared/src/models/utils/cater-for.ts

```diff
@@ -1,17 +1,35 @@
-import { IsNumber, IsString } from 'class-validator';
+import { IsNumber, IsString, Min } from 'class-validator';
 import { getCreate } from './get-create';
 
 export class CanCaterFor {
+    /**
+     * if {@link logicalOperator}  `=== 'and'` it is cumulative,
+     * how many red patients can the catering treat
+     * also affects the number of possible {@link yellow} and {@link green} patients
+     * that can be treated.
+     */
     @IsNumber()
+    @Min(0)
```

Comment:
It would be, yes.
But may be quite complicated to migrate (?), as whole history in exercises could change.

Reply:
We don't need to write a migration for them because I assumed that there was no exercise before where these values were a different value smaller than `0`.

There are other breaking changes that are non-trivial to migrate...